pages/services.html

```diff
@@ -20,3 +20,18 @@
 </div>
 
 <p>More services are on the way. If there's something that would be super beneficial for us to run, don't hesitate to reach out and ask. And of course, if you like our services, please consider <a href="/donate/">donating to support our server costs</a>, <em>any donation helps!</em></p>
+
+<h3>A brief note</h3>
+
+<p>We are providing these services because we believe in decentralized and federated networks, and free and open source software. The tools we host are a great and convenient entry-point to these services like Matrix, Mastodon, and WriteFreely. We hope that we are able to spur the adoption of these alternative networks, and we strongly believe all of our services are far better than traditional providers, such as Twitter, Facebook, and Google.</p>
+<p>However, as with <em>any</em> hosted, online service, remember that using our services involves risks that may or may not be acceptable to you. Using online services requires trust in the service provider. We don't know of any surefire way to evaluate a provider for privacy, or provide a service that we can guarantee is operating in a privacy-respecting way. Of course we strive to provide the best services possible for our community, but at the end of the day this involves you trusting us and the system administrators that host our services.</p>
+<p>As always, <strong>never</strong> trust a service provider with sensitive information, even if they are trustworthy.</p>
+<p>We also recommend you don't centralize all your online activity around a single provider's services (such as ours). This is because it is not a good idea to give a single party control over everything you do online.</p>
+<p>We highly recommend hosting your own services whenever possible. Not only is it a fantastic learning experience, but it allows you to have services entirely in your control. There are a number of great resources available, including...</p>
+<ul>
+  <li><a href="https://talk.libreho.st/">The libreho.st forum</a></li>
+  <li><a href="https://homebrewserver.club/">homebrewserver.club</a></li>
+  <li><a href="https://www.reddit.com/r/selfhosted/">r/selfhosted</a></li>
+</ul>
+<p>You can also join us on Matrix at <code>#general:privacytools.io</code> and ask around! We are always happy to help you start hosting your own services.</p>
+<p>The services we provide here are for the benefit of our community, and we collect as little data as possible. But it is a convenience more than a recommendation. You should always avoid trusting others with your information, and always encrypt.</p>
```

Comment:
```suggestion
<p>The services we provide here are for the benefit of our community, and we collect as little data as possible. But it is a convenience more than a recommendation. You should always avoid trusting others with your information, and encrypt everything.</p>
```

I think the last phrase is badly formatted and I am not sure how to format it better and this isn't improving much and I wonder if it should say "encrypt everything private" but that makes some ask what is the line between private and public (such as `#general:privacytools.io` FAQ)